recipes/eigency/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "eigency" %}
+{% set version = "2.0.0a1" %}
+{% set eigenversion = "3.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ eigenversion }}.{{ version }}
+
+source:
+  url: https://github.com/wouterboomsma/eigency/archive/refs/tags/2.0.0a1.tar.gz
```

```suggestion
  url: https://github.com/wouterboomsma/eigency/archive/refs/tags/{{ version }}.tar.gz
```
We have a bot that will update the recipe for new releases, but only if you use jinja here.